test_util/cluster.py

```diff
@@ -444,7 +400,8 @@ def wait_for_mesos_metric(cluster, host, key, value):
         run_bootstrap_nginx(bootstrap_host_tunnel, cluster.ssher.home_dir)
 
     upgrade_ordering = [
-        ('master', 'master', master_upgrade_order(cluster)),
+        # Upgrade masters in a random order.
+        ('master', 'master', random.sample(cluster.masters, len(cluster.masters))),
         ('slave', 'agent', cluster.agents),
         ('slave_public', 'public agent', cluster.public_agents),
     ]
```

Comment:
Could we log the master upgrade order here so that it's easier to debug? (I know it'll get logged as we go through it but would be nice to see up front)